x-pack/plugins/observability_shared/public/components/tags_list/tags_list.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiText, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EuiBadgeProps } from '@elastic/eui/src/components/badge/badge';
+
+export interface TagsListProps {
+  onClick?: (tag: string) => void;
+  tags?: string[];
+  noOfTagsToDisplay?: number;
+  color?: EuiBadgeProps['color'];
+  ignoreEmpty?: boolean;
+}
+
+const getFilterLabel = (tag: string) => {
+  return i18n.translate('xpack.observabilityShared.getFilterLabel.filter', {
+    defaultMessage: 'Click to filter list with tag {tag}',
+    values: {
+      tag,
+    },
+  });
+};
+
+const TagsList = ({
+  ignoreEmpty,
+  tags,
+  noOfTagsToDisplay = 3,
```

Comment:
I didn't get the `no` prefix!

```suggestion
  numberOfTagsToDisplay = 3,
```